### DISCUSSION TITLE
How about schedule a new release since v3.3.0 was 2021 build?

### DISCUSSION THREAD

**Discussion by dingtianran** [airbnb/lottie-ios#1625] [Q&A] [answered]
## Which Version of Lottie are you using?
Lottie 3.3.0

## Expected Behavior 
Nothing wrong

## Actual Behavior
All good but we want new code

**Comment by wiesener** [comment]
Any news here?

**Comment by calda** [comment, marked as answer]
Lottie 3.4.0 is now available: https://github.com/airbnb/lottie-ios/discussions/1627
